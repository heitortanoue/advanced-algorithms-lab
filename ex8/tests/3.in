9876543210 5
0123456789 2
