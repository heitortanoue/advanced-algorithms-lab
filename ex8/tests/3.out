9876543210
9823456710
